### ISSUE TITLE
UpdatePropertyByRef does not update Status

### ISSUE THREAD

**Issue by marseneault** [HomeSeer/Plugin-SDK#126]
**Environment and System Config**
- OS: [Windows]
- Language: [C#]
- IDE: [VS2019]
- HomeSeer Version: [v4.1.7.0]
- PluginSDK Version: [v1.2.1 nuget]

My plugin needs to update both the Device Value and the Device String (now known as Status).  The problem is that I also need to create Events that trigger on the value change of my devices and using both UpdateFeatureValueByRef and UpdateFeatureValueStringByRef in my plugin to update both the Value and the String results in my events firing off twice successively!

According to the documentation, using UpdatePropertyByRef can update the device without any Homeseer processing.  Using UpdatePropertyByRef(ref, EProperty.Status, "Test") does not update the Status string at all in Homeseer.

**Steps to Reproduce**
Using UpdateFeature, my events trigger twice but both the Value and the Status are updated - as expected, I guess?
```
HsFeature feature = HomeSeerSystem.GetFeatureByAddress("TestFeature");  
// Triggers events created with "just had its value set or changed".  
HomeSeerSystem.UpdateFeatureValueByRef(feature.Ref, 100);  
// Again, Triggers events created with "just had its value set or changed".  
HomeSeerSystem.UpdateFeatureValueStringByRef(feature.Ref, "Full blast");
```

Using UpdateProperty, the String (Status) **does not** get set but the event only triggers once
```
HsFeature feature = HomeSeerSystem.GetFeatureByAddress("TestFeature");
// Triggers events created with "just had its value set or changed".
HomeSeerSystem.UpdateFeatureValueByRef(feature.Ref, 100);
// The Feature String (Status) does not get updated in Homeseer - no errors reported by PluginSDK
HomeSeerSystem.UpdatePropertyByRef(feature.Ref, EProperty.Status, "Full blast");
```

**Expected behavior**
I need a way to update both the String (Status) and Value without having Homeseer process both changes.  As per the documentation, UpdatePropertyByRef is the alternative for UpdateFeatureValueStringByRef if you do not want Homeseer to trigger events but it does not actually update the device.

I've got many HS3 plugins where this was easily done with DeviceString and DeviceValue setting False to one in order to avoid the extra processing.

Is there another way? Or is this the correct approach but it does not work?

**Comment by spudwebb** [comment]
There is a bug in UpdatePropertyByRef  when used with EProperty.Status or EProperty.Value. It will be fixed in the next version of HS4 (probably 4.1.10.0)

This is the correct approach unless the status you want to set for the value 100 is always the same, in this case it's better to add a StatusGraphic to your feature for value 100 with a Label set to the status text. Then UpdateFeatureValueByRef(ref, 100) would set both value and status at the same time.

**Comment by marseneault** [comment]
Thank you, I'll be watching for the update then.
